File: test_coverage_lib.py
"""Library for testing type coverage reporting.
Has 5 functions but the app only calls 3 of them.
"""


def add(a, b):
    return a + b


def subtract(a, b):
    return a - b


def multiply(a, b):
    return a * b


def divide(a, b):
    return a / b


def modulo(a, b):
    return a % b

File: test-coverage-lib.js
/**
 * Library for testing type coverage reporting.
 * Has 5 functions but the app only calls 3 of them.
 */

function add(a, b) {
  return a + b;
}

function subtract(a, b) {
  return a - b;
}

function multiply(a, b) {
  return a * b;
}

function divide(a, b) {
  return a / b;
}

function modulo(a, b) {
  return a % b;
}

module.exports = { add, subtract, multiply, divide, modulo };
